test-framework/sudo-compliance-tests/src/path_search.rs

```diff
@@ -80,3 +80,37 @@ fn ignores_path_for_qualified_commands() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+#[ignore = "gh561"]
+fn paths_are_matched_using_realpath_in_sudoers() -> Result<()> {
+    let env = Env(["ALL ALL = /bin/true"]).build()?;
+
+    // this test assumes /bin is a symbolic link for /usr/bin, which is the
```

Comment:
to not make these tests not depend on the OS you could create a symlink to `/usr/bin` in, e.g., `/tmp/bin` (and resolve where `true` is located or create the shell script you'll run in a known location)